submissions/madmaxWMFU/frogger-game/js/app.js

```diff
@@ -0,0 +1,95 @@
+const playerStartValue = {
+    "x": 200,
+    "y": 400,
+    "icon": "images/char-boy.png"
+};
+
+let score = document.querySelector(".user-score");
+
+class Enemy {
+    constructor(x, y, speed) {
+        this.x = x;
+        this.y = y;
+        this.sprite = "images/enemy-bug.png";
+        this.speed = speed;
+        this.player = player;
+    }
+    
+    update(dt) {
+        this.x += this.speed * dt;
+        if(this.x > 510) {
+            this.x = -50;
+            this.speed = randomSpeed();
```

Comment:
`randomSpead()` - is a method for `Enemy` class. So, we can define them in class. Also methods must be named with verb. `getRandomSpead()` looks like better.